storage/src/main/java/io/delta/storage/GCSLogStore.java

```diff
@@ -57,7 +57,7 @@
  * This class is not meant for direct access but for configuration based on storage system.
  * See https://docs.delta.io/latest/delta-storage.html for details.
  */
-@Unstable
+@InterfaceStability.Unstable
```

Comment:
actually ... can you remove this?

sorry, the whole point of us moving GCSLogStore to /storage instead of /contribs is that it has been tried-and-tested by enough open source users for us to consider it production ready